Dockerfile

```diff
@@ -15,9 +15,12 @@ LABEL "maintainer"="https://github.com/laminas/technical-steering-committee/"
 
 WORKDIR /app
 
-RUN apk add --no-cache git gnupg libzip \
+RUN apk add --no-cache git gnupg libzip icu-dev \
     && apk add --no-cache --virtual .build-deps libzip-dev \
     && docker-php-ext-install zip \
+    && docker-php-ext-install bcmath \
+    && docker-php-ext-configure intl \
```

Comment:
Are Intl/ICU avoidable? None of the contents of this library really care about i18n

Reply:
psl requires intl and bcmath, intl ext requires icu.

psl doesn't do checks like `if(extension_loaded('intl'))`, but requires all extensions in `composer.json`.

Reply:
the fact that you didn't notice it before proves to me that it's the right things to do: people most likely have them locally.

Reply:
Works for me - just unfortunate having to add this stuff to the image, heh.